TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Teleop/IntoTheDeepTeleOp.java

```diff
@@ -99,32 +99,33 @@ public void loop(){
 
         // open claw
         if (gamepad2.a) {
-            claw.setPosition(0.75);
+            claw.setPosition(0.30);
         // close claw
         } else if (gamepad2.x) {
             claw.setPosition(0.25);
         }
 
-        // forward arm rotation (toward floor)
+        // forward slide rotation (toward floor)
         if (gamepad2.dpad_right) {
             armLeftFront.setPosition(armPosition.position);
             armRightFront.setPosition(1 - armPosition.position);
-            armPosition.update(0.005);
-        // backward arm rotation
+            armPosition.update(0.002);
+        // backward slide rotation (Up position)
         } else if (gamepad2.dpad_left) {
             armLeftFront.setPosition(armPosition.position);
             armRightFront.setPosition(1 - armPosition.position);
-            armPosition.update(-0.005);
+            armPosition.update(-0.002);
         }
 
-        // slides go up (must hold button to hold slide position)
+        // slides extend up (must hold button to hold slide position)
         if (gamepad2.dpad_up) {
             slideExtensionMotor.setPower(1.0);
-        // slides go down (must hold button to hold slide position)
+        // slides extend down (must hold button to hold slide position)
         } else if (gamepad2.dpad_down) {
             slideExtensionMotor.setPower(-1.0);
         } else {
             slideExtensionMotor.setPower(0);
+            slideExtensionMotor.setZeroPowerBehavior(DcMotor.ZeroPowerBehavior.BRAKE);
```

Comment:
This should happen at the top, not here. It's a general setting, not a command ya feel